### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by thesocialdev**
1) Add i18next to the react app
2) Make WikidataTypeAhead aware of language and request Wikidata using
browser canonical language
3) Make wikidata endpoint aware of user's language and forward it to mwapi
4) Make personality endpoint aware of user's language

### CODE REVIEW

File: server/lib/wikidataResolver.js [hunk 1]
```diff
@@ -1,24 +1,29 @@
 const axios = require("axios");
 
 module.exports = class WikidataResolver {
-    async fetchProperties(wikidataId) {
+    async fetchProperties(params) {
+        console.log(params);
```
Comment (wellcda) [inline server/lib/wikidataResolver.js:5]: @thesocialdev

File: server/lib/wikidataResolver.js [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
+            params.language
+        );
     }
 
     async extractProperties(wikidata, lang = "en") {
         const wikidataProps = {};
         if (!wikidata) {
             return {};
         }
+        console.log(lang);
```
Comment (wellcda) [inline server/lib/wikidataResolver.js:26]: @thesocialdev

**Comment by wellcda** [comment]
@thesocialdev could you make it up to date with the stage branch, please?

**Comment by thesocialdev** [comment]
> Please resolve conflicts

Done. Thanks for reviewing it!

**Pull request merged**